src/less/index.less

```diff
@@ -0,0 +1,17 @@
+// Clever Styles
+// Version 0.6.2
+
+// Color and font variables
+@import "./colors.less";
+@import "./fonts.less";
+@import "./type-size.less";
+@import "./grid.less";
+
+// Component Modules
+@import "../Button/Button";
+@import "../ConfirmationButton/ConfirmationButton";
+@import "../SegmentedControl/SegmentedControl";
+@import "../TextInput/TextInput";
+@import "../CopyableInput/CopyableInput";
+@import "../Modal/Modal";
```

Comment:
alternatively we can use [less-plugin-glob](https://github.com/just-boris/less-plugin-glob) to just discover less files to import.